openstates/scrape/base.py

```diff
@@ -188,10 +191,39 @@ def save_object(self, obj):
                 upload_file_path = file_path[
                     file_path.index("_data") + len("_data") + 1 :
                 ]
+                # This sets the upload file path to be within a bill folder within a session folder within a state folder 
+                upload_file_path = upload_file_path[:3] + obj.legislative_session + '/' + obj.identifier + '/' + upload_file_path[3:]
```

Comment:
Let's use formatted strings here instead - much cleaner/easier to read.
And then if you move `jurisdiction = upload_file_path[:3]` higher up, you can use it here and we can remove the comment as it should be fairly self explanatory at that point